org.tug.texworks.yaml

```diff
@@ -14,7 +14,6 @@ finish-args:
   - --filesystem=host
   - --env=TEXMFCACHE=$XDG_CACHE_HOME
   - --env=SESSION_MANAGER= # required to avoid Qt warning
-  - --env=PATH=/app/texlive/bin/x86_64-linux:/app/texlive/bin/aarch64-linux:/app/bin:/usr/bin # required to find texlive binaries
 add-extensions:
   org.freedesktop.Sdk.Extension.texlive:
```

Comment:
the key problem is this approach to the use of the texlive extension. One of the solution would be to make this the standard way to use it, everywhere, instead of the bulk copy to /app.